docs/JOINMARKET-QT-GUIDE.md

```diff
@@ -74,6 +74,17 @@ stick to that convention.
 
 The wallet will now automatically load from the your Bitcoin Core node. It may take a few seconds, during which you'll see "Reading wallet from blockchain...".
 
+If JoinMarketQt stucks in "Reading wallet from blockchain..." state, close it. 
```

Comment:
I'm pretty sure there are other reasons for this bug/error condition where it just hangs, and also: it should not hang with a default wallet (Bitcoin Core will use `wallet.dat` by default). So the advice you're giving below is a recommendation (even, a very strong one), but it should not be needed to avoid this error condition.

Another thing we should add to the notes, since it crops up from time to time, is that Bitcoin Core should not be compiled without wallet support (this is a common error from more advanced users who compile Core themselves).

Reply:
OK, in absence of response on that, what I think I'll do is:
add an extra commit after this one: add the point about compilation, plus changing the wording of your part to explain that this is *one* possible cause of 'stuckness' that you describe, and also to make the nondefault Core wallet a "strongly recommended".